core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -442,6 +443,32 @@ class RelWriterTest {
       + "  ]\n"
       + "}";
 
+  public static final String LIMIT_DYNAMIC_PARAM = "{\n"
```

Comment:
given that this is used only once my personal preference would be to make it a local variable to the function that uses it. But that's not a big deal.

Reply:
Thank you for your suggestion.